src/Main.elm

```diff
@@ -86,6 +90,7 @@ init location =
                         Date.compare (watchDate m1) (watchDate m2)
                     )
         , selectedMovie = NotSelected
+        , focusedMovie = NotSelected
```

Comment:
Given how the UI works, do we still need `selectedMovie`? Drawing a movie from the hat could just set the movie as `focusedMovie` and remove it from `unwatched`, I think.

Reply:
Yep. I had kept it as a way to fall back in case the load failed, but I worked that out, so I believe I can remove it now.